src/markdown/markdown-body.scss

```diff
@@ -78,6 +78,12 @@
     border: 0;
   }
 
+  td {
```

Comment:
Should we move this into [markdown/tables.scss](https://github.com/primer/css/blob/c22d30b48819eb74510ae5ce6ce5ffcfbc4814a8/src/markdown/tables.scss)? Since it's a specific override for tables.

Reply:
Ah, I missed that file! Yep, that makes sense – styles have been moved.